eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -8,8 +8,11 @@
           "exclude": [
             "com\\.azure\\..+\\.implementation(\\..+)?",
             "com\\.fasterxml\\.jackson\\..+",
-            "reactor\\.netty\\..+",
-            "io\\.netty\\..+"
+            "reactor\\..+\\..+",
+            "io\\.netty\\..+",
+            "org\\.springframework\\..+",
+            "com\\.nimbusds(\\..+)?",
+            "com\\.microsoft\\.azure\\.spring\\..+"
```

Comment:
As we move from using track one to track two, should the package naming be updated to, e.g. com.azure.spring?

Reply:
We'll move to `com.azure` in our next major version and keep using `com.microsoft.azure` for now.